src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -273,6 +274,8 @@ private void getUserProfile() {
 		currentUserBundle = null;
 		int mask = PROFILE | ORC_ID;
 		Long currentUserIdLong = currentUserId != null ? Long.parseLong(currentUserId) : null;
+		// capture current state of user profile editor widget, to restore after reconfiguring with the current user profile.
+		boolean isEditing = getUserProfileEditorWidget().isEditingMode();

```

Comment:
When we update the UserBundle, remember if the profile widget is in edit mode, and restore after we reconfigure.